go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openstack-k8s-operators/octavia-operator
 
-go 1.20
+go 1.21
```

Comment:
This is added by `go mod tidy`. I tried to install go 1.20, but it didn't work. I'm not sure if this causes issues.